src/image/image.h

```diff
@@ -77,8 +77,18 @@ class Image {
   uint8_t *GetPixelData(size_t x, size_t y);
   const uint8_t *GetPixelData(size_t x, size_t y) const;
 
+  double *GetValue(size_t x, size_t y) ;
```

Comment:
Let's use `GetXXXData` for acquiring pointers, and `GetXXX` for acquiring values. Everything else looks nice!